release/docker/README.md

```diff
@@ -36,3 +36,12 @@ For further details, please consult the mitmproxy [documentation](http://docs.mi
 # Tags
 
 The available release tags can be seen [here](https://hub.docker.com/r/mitmproxy/mitmproxy/tags/).
+
+* `master` always tracks the git-master branch and represents the unstable development tree.
+* `latest` always points to the same image as the most recent stable release, including bugfix releases (e.g., `4.0.0` and `4.0.1`).
+* `X.Y.Z` tags contain the mitmproxy release with this version number.  
+  - All packages, dependencies, and the base image that were available at the time of the mitmproxy release.
```

Comment:
We have the following at https://docs.mitmproxy.org/stable/overview-installation/#self-contained-pre-built-binary-packages:

> Dependencies in the binary packages are frozen on release, and can’t be updated in situ. This means that we necessarily capture any bugs or security issues that may be present. We don’t generally release new binary packages simply to update dependencies (though we may do so if we become aware of a really serious issue). If you use our binary packages, please make sure you update regularly to ensure that everything remains current.

Do we maybe want to re-use the same wording for Docker (s/binary packages/docker images/)?